README.md

```diff
@@ -1,34 +1,57 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
+## General overview of what this repo contains
+
+This repo contains a frontend interface built with nextJS, the UI connects to:
+- [GemFarm](https://github.com/gemworks/gem-farm) for Staking as a Service through a service layer API built by Honey Finance - WIP
+- [Honey-SDK](https://github.com/honey-labs/honey-sdk) for permissionless lending pools that use NFTs as collateral - WIP
```

Comment:
add "the documentation about Goki wallet https://github.com/GokiProtocol/walletkit
and degen-ui https://degen-xyz.vercel.app/ "

Reply:
good addition! will add it tomorrow.